main/src/main/scala/sbt/Cross.scala

```diff
@@ -323,7 +323,7 @@ object Cross {
         if (included.isEmpty) {
           sys.error(
             s"""Switch failed: no subprojects list "$version" (or compatible version) in crossScalaVersions setting.
-               |If you want to force it regardless, call ++ $version!""".stripMargin
+               |If you want to force it regardless, call ++ $version""".stripMargin
```

Comment:
This slug should be undone. This occurrence of `!` is on a log statment not an actual usage.